cmd/admin-info.go

```diff
@@ -230,12 +226,19 @@ func (u clusterStruct) String() (msg string) {
 			english.Plural(int(u.Info.Buckets.Count), "Bucket", ""),
 			english.Plural(int(u.Info.Objects.Count), "Object", ""))
 	}
-	if backendType != "FS" {
+	if backendType != madmin.FS {
 		// Summary on total no of online and total
 		// number of offline disks at the Cluster level
-		msg += fmt.Sprintf("%s online, %s offline\n",
-			english.Plural(totalOnlineDisksCluster, "drive", ""),
-			english.Plural(totalOfflineDisksCluster, "drive", ""))
+		bkInfo, ok := u.Info.Backend.(madmin.ErasureBackend)
+		if ok {
+			msg += fmt.Sprintf("%s online, %s offline\n",
+				english.Plural(bkInfo.OnlineDisks, "drive", ""),
+				english.Plural(bkInfo.OfflineDisks, "drive", ""))
+		} else {
+			msg += fmt.Sprintf("%s online, %s offline\n",
```

Comment:
Printing disks offline/online does not work for gateway. I think we don't have to print anything if this is not erasure.

Reply:
How does gateway come into picture here @vadmeste 

This code never prints in Gateway. Did you test it with gateway?

Reply:
done fixed @vadmeste